python/Pipfile

```diff
@@ -10,6 +10,7 @@ mypy = "*"
 [packages]
 numpy = "*"
 pylint = "*"
```

Comment:
Is this actually needed in the release version or could it be moved to dev?

Reply:
probs could be moved to dev